tests/suites/0_stateless/06_show/06_0003_show_settings.result

```diff
@@ -3,6 +3,7 @@ enable_new_processor_framework	1	1	SESSION	Enable new processor framework if val
 enable_planner_v2	0	0	SESSION	Enable planner v2 by setting this variable to 1, default value: 0	UInt64
 field_delimiter	,	,	SESSION	Format field delimiter, default value: ,	String
 flight_client_timeout	60	60	SESSION	Max duration the flight client request is allowed to take in seconds. By default, it is 60 seconds	UInt64
+group_by_two_level_threshold	10000	10000	SESSION	The threshold of keys to open two-level aggregation, default value: 10000	UInt64
```

Comment:
Need another tests  to enable this `group_by_two_level_threshold`.

Example:

```
set group_by_two_level_threshold = 10;

select ....from numbers(...)   group by ...
```

Reply:
Ok!